pkg/apis/walrus/v1/environment.go

```diff
@@ -25,38 +26,12 @@ type Environment struct {
 
 var _ runtime.Object = (*Environment)(nil)
 
-// EnvironmentType describes the type of environment.
-// +enum
-type EnvironmentType string
-
-const (
-	// EnvironmentTypeDevelopment means the environment is for development.
-	EnvironmentTypeDevelopment EnvironmentType = "development"
-	// EnvironmentTypeStaging means the environment is for staging.
-	EnvironmentTypeStaging EnvironmentType = "staging"
-	// EnvironmentTypeProduction means the environment is for production.
-	EnvironmentTypeProduction EnvironmentType = "production"
-)
-
-func (in EnvironmentType) String() string {
-	return string(in)
-}
-
-func (in EnvironmentType) Validate() error {
-	switch in {
-	case EnvironmentTypeDevelopment, EnvironmentTypeStaging, EnvironmentTypeProduction:
-		return nil
-	default:
-		return errors.New("invalid environment type")
-	}
-}
-
 // EnvironmentSpec defines the desired state of Environment.
 type EnvironmentSpec struct {
 	// Type is the type of the environment.
 	//
 	// +k8s:validation:enum=["development","staging","production"]
-	Type EnvironmentType `json:"type"`
+	Type walruscore.EnvironmentType `json:"type"`
```

Comment:
`+k8s:validation:enum=["development","staging","production"]`, should use the same value as walruscore.EnvironmentType